metadetect/lsst_metadetect.py

```diff
@@ -84,26 +86,25 @@ def _measure(self, mbobs, shear_str):
 
         res = self._fitter.go(mbobs_list)
         if res is not None:
-            res = self._add_positions_and_psf(medsifier, res, shear_str)
+            res = self._add_positions_and_psf(medsifier, res, shear_str, mbobs_list)
 
         return res
 
-    def _add_positions_and_psf(self, medsifier, res, shear_str):
+    def _add_positions_and_psf(self, medsifier, res, shear_str, mbobs_list):
         """
         TODO add positionis etc.
         """
 
         new_dt = [
-            ('row', 'f4'),
-            ('col', 'f4'),
-            ('row_noshear', 'f4'),
-            ('col_noshear', 'f4'),
+            ('box_size', 'i4'),
+            ('row0', 'i4'),  # bbox row start
+            ('col0', 'i4'),  # bbox col start
+            ('row', 'f4'),  # row in image. Use row0 to get to global pixel coords
+            ('col', 'f4'),  # col in image. Use col0 to get to global pixel coords
+            ('row_noshear', 'f4'),  # noshear row
+            ('col_noshear', 'f4'),  # noshear col
```

Comment:
We can and should normalize our conventions on this. I'd like to keep the same names in both the DES and LSST codes if possible. Plus I did a bunch of renaming in the DES mdet codes which maybe should be ported here. I think this may need another PR to fix.

Reply:
I started appending `_det` to the detected positions and removing `_noshear` from the unsheared ones. This makes things a bit easier to parse downstream.